config/config.ts

```diff
@@ -0,0 +1,4 @@
+import { readFileSync } from 'fs';
```

Comment:
nit: this file can be be under `/src/config.ts`. IMO the `/config` directory should only contain raw config data, how the app reads/interprets that config data should be under the app's own directory aka `/src`